src/guiapi/src/ScreenShot.cpp

```diff
@@ -2,13 +2,18 @@
 //#include <string.h>
 #include <unistd.h>
 #include "ScreenShot.hpp"
-#include "st7789v.hpp"
 #include "display.h"
 #include <inttypes.h>
 #include <dirent.h>
 #include <stdio.h>
 #include <unistd.h>
 
+#ifdef USE_ST7789
+static const uint8_t bytes_per_pixel = 2;
```

Comment:
With this on lines 21-22 we can remove ST7789V_BYTES_PER_PIXEL.
Similarly on 48-52 we can replace it with bytes_per_pixel which makes it much cleaner.

Reply:
Also in the block of lines 65-90

Reply:
Fixed